public/assets/js/scripts.js

```diff
@@ -322,101 +333,117 @@ $(document).ready(function () {
+    }
+    if (immediatePrivate || immediateFederal || spreadFederal || spreadPrivate) {
+      output.push(categories.mortgage);
+    }
+    if (directSpread) {
+      output.push(categories.spread);
+    }
+    if (rentToOwn) {
+      output.push(categories.rentToOwn);
     }
     if (hybrid) {
-      let package = {
-        title: `Hybrid <img src="./assets/img/icons/question-mark.svg" alt="payment">`,
-        message: 'Start BALLing with an initial deposit of 1%'
-      }
-      output.push(package);
+      output.push(categories.hybrid);
     }
-    
-    if ($(window).width() < MOBILE_WIDTH) {
-      for (let i = 0; i < output.length; i++) {    
+
+    for (let i = 0; i < output.length; i++) {
+      if ($(window).width() < MOBILE_WIDTH) {
         recommendationBody += `<div class="card recommendation-card">
-                                <div class="card-header" id="heading${i}">
-                                  <a data-toggle="collapse" href="#collapse${i}" aria-expanded="false" aria-controls="collapse${i}">
-                                    <h6 class="mb-0">
-                                      ${output[i].title}
-                                      <span class="recommendation-card-icon">&nbsp; ⌄ </span>
-                                    </h6>
-                                  </a>
-                                </div>
-                                <div id="collapse${i}" class="collapse" aria-labelledby="heading${i}" data-parent="#accordion">
-                                  <p class="heading">Requirements</p>
-                                  <div class="card-body">${output[i].message}</div>
-                                  <p class="target">Target:</p>
-                                  <h4 class="price">NGN ${formatToCurrency(avgPropertyCost)}</h4>
-                                </div>
-                              </div>`;
-      }
-      recommendation = `<div id="accordion">
-                          ${recommendationBody}
-                        </div>`;
-    } else {
-      for (let i = 0; i < output.length; i++) {
+                                  <div class="card-header recommendation-card-header" id="heading${i}">
+                                    <a data-toggle="collapse" href="#collapse${i}" aria-expanded="false" aria-controls="collapse${i}">
+                                      <h6 class="mb-0">
+                                        ${output[i].title}
+                                        <img src="./assets/img/icons/question-mark.svg" alt="payment">
+                                        <span class="recommendation-card-icon">&nbsp; ⌄</span>
+                                      </h6>
+                                    </a>
+                                  </div>
+                                  <div id="collapse${i}" class="collapse" aria-labelledby="heading${i}" data-parent="#accordion">
+                                    <p class="heading">Requirements</p>
```

Comment:
Can we use a more personalized/ customized class, `heading` seems too generic and can easily be overwritten with other CSS classes. Most of the classes (target, price) should be prefixed (maybe with recommendation-accordion, so the heading class will become recommendation-accordion-heading) to prevent accidental overwrite from other CSS classes.